.pipelines/templates/checkout-upstream-repo.yaml

```diff
@@ -0,0 +1,4 @@
+steps:
+  - bash: |
+      git clone -b v${CHART_TAG} $(upstream.repo) $(System.DefaultWorkingDirectory)/osm
```

Comment:
Should this be `$(System.DefaultWorkingDirectory)/openservicemesh` to match the repo?

Reply:
https://github.com/Azure/osm-azure/pull/38#discussion_r568896100